bundles/org.openhab.binding.evcc/src/main/java/org/openhab/binding/evcc/internal/EvccHandler.java

```diff
@@ -150,8 +148,9 @@ public void initialize() {
                         updateStatus(ThingStatus.ONLINE);
                         refreshOnStartup();
                     } catch (Exception e) {
-                        updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.OFFLINE.CONFIGURATION_ERROR,
-                                "Failed to connect to evcc: " + e);
+                        logger.debug("Initialization failed:", e);
+                        updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.HANDLER_INITIALIZING_ERROR,
```

Comment:
**HANDLER_INITIALIZING_ERROR** is not correct for status **OFFLINE**, this should only be set by the framework.

Reply:
As thus us not needed, I will completely remove it.